sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/second/SecondFragment.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.sample.compose.ui.screens.second
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.fragment.navArgs
+import co.nimblehq.sample.compose.ui.base.BaseComposeFragment
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SecondFragment : BaseComposeFragment() {
```

Comment:
Shall we rename it to `DetailFragment`, same as `DetailScreen`

Reply:
Conclusion: Let's use SecondFragment and SecondScreen for now to align with XML. We can consider renaming them later